frontend/src/components/ExperimentCollection/ExperimentCollection.scss

```diff
@@ -220,7 +220,14 @@
                         padding: 0 15px 15px;
                         opacity: 0.8;
                     }
+                    .counter {
+                        color: white;
+                        position: relative;
+                        top: 30vh;
```

Comment:
Isn't `vh` a height relative to the viewport? What happens if you resize the screen?